udata_hydra/datalake_service.py

```diff
@@ -94,13 +104,20 @@ async def process_resource(url: str, dataset_id: str, resource_id: str, response
                         "location": storage_location,
                         "mime_type": mime_type,
                         "filesize": filesize,
+                        "delimiter": delimiter,
```

Comment:
yes because udata-analysis-consumer is consuming `resource.stored` topic, not `analysed`. And this delimiter is used there. But we can maybe copy these infos (delimiter and encoding) in `analysed` topic. WDYT ?

Reply:
let's keep it that way for now.